### ISSUE TITLE
Traking

### ISSUE THREAD

**Issue by Hackrfan** [SatDump/SatDump#686]
traking
Question: is there a way to limit the satellite list in the traker to only the required satellites ( faforites ).
The amount of satellites in the selection list is much too long in my opinion. A limitation or selection would be very helpful.
I may have overlooked something here.

**Comment by Aang23** [comment]
Is the search not usable in your instance?

**Comment by Hackrfan** [comment]
Yes, of course, the search function is available, but it would be easier to have a favorites list here. I need a maximum of 10 satellites from the selection window.
